### PR TITLE
DM: Add v1.0.3 release notes and update DM upgrade doc

### PR DESCRIPTION
**Pull request by ericsyh**
### What is changed, added or deleted? 

Add DM v1.0.3 release notes and update DM upgrade doc

### Which version does your change affect? 

dev/v3.0/v3.1/v2.0

### CODE REVIEW

**Comment by ericsyh** [comment]
@WangXiangUSTC @csuzhangxc @TomShawn PTAL

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 1]
```diff
@@ -14,6 +14,37 @@ category: reference
...
+
+- dmctl 支持命令式使用
+- 支持同步 `ALTER DATABASE` DDL 语句
+- 优化 DM 错误提示信息
+- 修复全量导入模块在暂停或退出时 Data Race 导致 panic 的问题 
+- 修复对下游进行重试操作时，`stop-task` 和 `pause-task` 可能不生效的问题 
+
+### 升级操作示例
+
+1. 下载新版本 DM-Ansible, 确认 `inventory.ini` 文件中 `dm_version = v1.0.3`
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
1. 下载新版本 DM-Ansible，确认 `inventory.ini` 文件中 `dm_version = v1.0.3`
```

File: dev/reference/tools/data-migration/releases/1.0.3.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## 详细变更及问题修复
+
+- dmctl 支持命令式使用 [#364](https://github.com/pingcap/dm/pull/364)
+- 优化 DM 错误提示信息 [#351](https://github.com/pingcap/dm/pull/351)
+- 优化 `query-status` 命令输出内容 [#357](https://github.com/pingcap/dm/pull/357)
+- 优化 DM 不同任务类型的权限检查 [#374](https://github.com/pingcap/dm/pull/374)
+- 支持对重复引用的路由配置和过滤配置进行检查 [#385](https://github.com/pingcap/dm/pull/385)
+- 支持同步 `ALTER DATABASE` DDL 语句 [#389](https://github.com/pingcap/dm/pull/389)
+- 优化 DM 异常重试机制 [#391](https://github.com/pingcap/dm/pull/391)
+- 修复全量导入模块在暂停或退出时 Data Race 争导致 panic 的问题 [#353](https://github.com/pingcap/dm/pull/353)
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/releases/1.0.3.md]:
```suggestion
- 修复全量导入模块在暂停或退出时 Data Race 导致 panic 的问题 [#353](https://github.com/pingcap/dm/pull/353)
```

File: dev/reference/tools/data-migration/releases/1.0.3.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## 改进提升
+
+- dmctl 支持命令式使用
+- 支持同步 `ALTER DATABASE` DDL 语句
+- 优化 DM 错误提示信息
+
+
+## 问题修复
+
+- 修复全量导入模块在暂停或退出时 Data Race 导致 panic 的问题
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/releases/1.0.3.md]:
```suggestion
- 修复全量导入模块在暂停或退出时 data race 导致 panic 的问题 
```

File: dev/reference/tools/data-migration/dm-upgrade.md [hunk 4]
```diff
@@ -14,6 +14,37 @@ category: reference
...
+UTC Build Time: 2019-12-13 07:04:53
+Go Version: go version go1.13 linux/amd64
+```
+
+### 主要变更
+
+- dmctl 支持命令式使用
+- 支持同步 `ALTER DATABASE` DDL 语句
+- 优化 DM 错误提示信息
+- 修复全量导入模块在暂停或退出时 Data Race 导致 panic 的问题
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/dm-upgrade.md]:
```suggestion
- 修复全量导入模块在暂停或退出时 data race 导致 panic 的问题 
```

**Comment by csuzhangxc** [comment]
@ericsyh please fix the CI

**Comment by ericsyh** [comment]
@csuzhangxc @TomShawn PTAL again. All fixed

File: dev/reference/tools/data-migration/releases/1.0.3.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+- dmctl 支持命令式使用 [#364](https://github.com/pingcap/dm/pull/364)
+- 优化 DM 错误提示信息 [#351](https://github.com/pingcap/dm/pull/351)
+- 优化 `query-status` 命令输出内容 [#357](https://github.com/pingcap/dm/pull/357)
+- 优化 DM 不同任务类型的权限检查 [#374](https://github.com/pingcap/dm/pull/374)
+- 支持对重复引用的路由配置和过滤配置进行检查 [#385](https://github.com/pingcap/dm/pull/385)
+- 支持同步 `ALTER DATABASE` DDL 语句 [#389](https://github.com/pingcap/dm/pull/389)
+- 优化 DM 异常重试机制 [#391](https://github.com/pingcap/dm/pull/391)
+- 修复全量导入模块在暂停或退出时 data race 导致 panic 的问题 [#353](https://github.com/pingcap/dm/pull/353)
+- 修复对下游进行重试操作时，`stop-task` 和 `pause-task` 可能不生效的问题 [#377](https://github.com/pingcap/dm/pull/377)
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/releases/1.0.3.md]: 替换为 `#400` ?

File: dev/reference/tools/data-migration/releases/1.0.3.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- 更新 Golang 版本至 1.13 以及其他依赖包版本 [#362](https://github.com/pingcap/dm/pull/362)
+- 过滤 SQL 执行时出现的 `context canceled` 错误[#382](https://github.com/pingcap/dm/pull/382)
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/releases/1.0.3.md]:
```suggestion
- 过滤 SQL 执行时出现的 `context canceled` 错误 [#382](https://github.com/pingcap/dm/pull/382)
```

File: dev/reference/tools/data-migration/releases/1.0.3.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+- 修复使用 DM-ansible 滚动升级 DM 监控过程中出错导致升级失败的问题[#408](https://github.com/pingcap/dm/pull/408)
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/releases/1.0.3.md]:
```suggestion
- 修复使用 DM-ansible 滚动升级 DM 监控过程中出错导致升级失败的问题 [#408](https://github.com/pingcap/dm/pull/408)
```

File: dev/reference/tools/data-migration/releases/1.0.3.md (hunk 5 above)
Comment (ericsyh) [reply dev/reference/tools/data-migration/releases/1.0.3.md]: Updated in https://github.com/pingcap/docs-cn/pull/2118/commits/26de354c0a90c1ea4b4ae2d0ec692df9aac41032

**Pull request merged**